tests/Unit/AutoDiscoveredValuesTraitTest.php

```diff
@@ -44,6 +45,11 @@ public function testThrowsOnChoicesMisuses()
 
         AutoDiscoveredEnumMisusingChoices::foo();
     }
+
+    public function testItAutoDiscoversValuesFromDumbEnum()
+    {
+        $this->assertSame(['foo', 'bar', 'baz'], AutoDiscoveredDumbEnum::values());
```

Comment:
I don’t really mind but since the discovered classes are `[self::class, DumbEnum::class]` I would have expected `'bar'` to be first